dash-pipeline/bmv2/dash_pipeline.p4

```diff
@@ -278,6 +277,7 @@ control dash_ingress(
             (meta.flow_sync_state == dash_flow_sync_state_t.FLOW_MISS &&
              hdr.packet_meta.packet_source == dash_packet_source_t.EXTERNAL))
         {
+            trusted_vni_stage.apply(hdr, meta);
```

Comment:
maybe we missed the inbound routing table for the VNI matching, if we simply return it here.

Reply:
Add a TODO comment and will revisit it after inbound routing HLD done.